extensions/resteasy-common/deployment/src/main/java/io/quarkus/resteasy/common/deployment/RestClientBuildItem.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.resteasy.common.deployment;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class RestClientBuildItem extends MultiBuildItem {
```

Comment:
Can we have some Javadoc on this one?

It seems pretty generic to be able to understand what it's used for

Reply:
Done.